testing/libuninameslist/APKBUILD

```diff
@@ -1,38 +1,38 @@
 # Contributor: Ed Robinson <ed@reevoo.com>
 # Maintainer: Ed Robinson <ed@reevoo.com>
 pkgname=libuninameslist
-pkgver=20170807
+pkgver=20190305
 pkgrel=0
 pkgdesc="A Library of Unicode names and annotation data"
 url="https://github.com/fontforge/libuninameslist"
 arch="all"
 license="BSD-3-Clause"
 source="https://github.com/fontforge/$pkgname/releases/download/$pkgver/$pkgname-dist-$pkgver.tar.gz"
-builddir="$srcdir/$pkgname-$pkgver"
 makedepends="autoconf automake libtool"
-subpackages="$pkgname-dev"
+subpackages="$pkgname-static $pkgname-dev $pkgname-doc"
 
 prepare() {
-        default_prepare
-
-        cd "$builddir"
-        autoreconf -i
-        automake --foreign
+	default_prepare
+	autoreconf -fi
 }
 
 build() {
-        cd "$builddir"
-        ./configure --prefix=/usr
-        make
+	./configure --prefix=/usr
+	make
 }
 
 check() {
-        cd "$builddir"
-        make check
+	make check
 }
 
 package() {
-        cd "$builddir"
-        make DESTDIR="$pkgdir" install
+	make DESTDIR="$pkgdir" install
+}
+
+static() {
+	depends=""
+	pkgdesc="$pkgdesc (static library)"
```

Comment:
maybe just call `default_static`?

Reply:
Didn't get greenlight from abuild devs to use the newly introduced default_static.